Backend/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Base image used  
+FROM node:alpine 
```

Comment:
Lets use the same base image as JasmineGraph project. See [URL](https://github.com/miyurud/jasminegraph-prerequisites/blob/master/Dockerfile#L1)

```
FROM ubuntu:jammy
```

Reply:
Since this is a Node-based project, it would be better to use a Node image as the base for optimal compatibility and size.

Reply:
Ok